src/components/Book.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 
 const Book = (book) => {
```

Comment:
- [ ] [OPTONAL] If you wanted to save lines of code you could already destructure the book object in the first line of the function.
Ex:
```javaScript
const Book = ({id, title, category, handleRemoveBook}) => {
```

Reply:
@uryelah Thank you for the feedback. Indeed, the restructuring is great, but I've been required by a previous TSE to send the book object.